docs/examples/entangled_states.ipynb

```diff
@@ -294,7 +277,16 @@
    "cell_type": "code",
    "execution_count": 8,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/var/folders/q_/6jnpzg_d225fj__b29k4wk200000gp/T/ipykernel_67350/1242378463.py:2: DeprecationWarning: Call to deprecated method get_qubits. (The indices flag will be removed. Use get_qubit_indices() instead.) -- Deprecated since version 4.0.\n",
```

Comment:
Can you show the recommended call approach so this deprecation warning is not emitted?

Reply:
Oops, fixed